plugin/teksi_wastewater/interlis/gui/interlis_export_settings_dialog.py

```diff
@@ -106,3 +112,10 @@ def selected_model(self):
 
     def selected_models(self):
         return self.export_model_selection_comboBox.currentData()
+
+    def selected_orientation(self):
+        # neu 23.4.2024 added float()
+        eorientation = float(self.export_orientation_selection_comboBox.currentText())
```

Comment:
```suggestion
        eorientation = self.export_orientation_selection_comboBox.currentData()
```

And than you can use [currentData](https://doc.qt.io/qt-5/qcombobox.html#currentData-prop) at this place